<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="en">
<context>
    <name>EditableGeoCsv</name>
    <message>
        <location filename="geocsv_plugin.py" line="57"/>
        <source>Add GeoCSV layer</source>
        <translation>Add GeoCSV layer</translation>
    </message>
    <message>
        <location filename="geocsv_plugin.py" line="64"/>
        <source>Editable GeoCSV</source>
        <translation>Editable GeoCSV</translation>
    </message>
</context>
<context>
    <name>GeoCsvNewController</name>
    <message>
        <location filename="geocsv_controller.py" line="117"/>
        <source>invalid file path</source>
        <translation>Error:  Invalid file path</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="176"/>
        <source>unknown csvt attribute: {}</source>
        <translation>Error: Unknown csvt attribute: {}</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="178"/>
        <source>csv&lt;-&gt;csvt missmatch</source>
        <translation>Notice: csv&lt;-&gt;csvt missmatch</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="180"/>
        <source>csvt geometry type exception</source>
        <translation>Error: Wrong geometry information detected in CSVT File. Please set again</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="182"/>
        <source>no csvt file found</source>
        <translation>Notice: No csvt file found</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="206"/>
        <source>error while loading csv</source>
        <translation>Error: An error occured while loading the csv file</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="196"/>
        <source>error in geometry selection</source>
        <translation>Error: No valid geometry information could be generated with selected fields. Please choose again</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="77"/>
        <source>GeoCSV Layer created</source>
        <translation>GeoCSV layer created</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="77"/>
        <source>The layer &quot;{}&quot; was created successfully.</source>
        <translation>The layer &quot;{}&quot; was created successfully.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="136"/>
        <source>Open GeoCSV File</source>
        <translation>Open GeoCSV file</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="307"/>
        <source>CSVT File Error</source>
        <translation>Notice: Error in CSVT file detected. Please set again</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="84"/>
        <source>The csvt file couldn&apos;t be updated on disk.</source>
        <translation>The csvt file couldn&apos;t be updated on disk.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="307"/>
        <source>An error occured while trying to update the CSVT file according to the new attribute types. Please update the csvt file manually.</source>
        <translation>An error occured while trying to update the CSVT file according to the new attribute types. Please update the csvt file manually.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="142"/>
        <source>not utf8 encoded</source>
        <translation>Error: The selected csv file is not utf-8 encoded.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="157"/>
        <source>error in converting</source>
        <translation>The conversion wasn&apos;t successful. Please convert to UTF-8 manually and try again.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="152"/>
        <source>backup created</source>
        <translation>Backup created</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="152"/>
        <source>Created backup on &quot;{}&quot;</source>
        <translation>Backup on &quot;{}&quot; created.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="154"/>
        <source>Converted</source>
        <translation>Converted</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="154"/>
        <source>Successfully converted to utf-8</source>
        <translation>CSV successfully converted to utf-8.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="136"/>
        <source>Files (*.csv *.tsv *.*)</source>
        <translation>Files (*.csv *.tsv *.*)</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="121"/>
        <source>invalid delimiter. expected &quot;{}&quot;</source>
        <translation>Erroneous delimiter detected. The delimiter should be a &quot;{}&quot;.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="82"/>
        <source>CSVT File created/updated.</source>
        <translation>CSVT-File erstellt/aktualisiert.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="82"/>
        <source>The CSVT file was successfully created/updated on disk.</source>
        <translation>The CSVT file has been created/updated.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="87"/>
        <source>PRJ File created.</source>
        <translation>PRJ File created.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="87"/>
        <source>The PRJ file was successfully created on disk.</source>
        <translation>The PRJ file has been created.</translation>
    </message>
</context>
<context>
    <name>GeoCsvReconnectController</name>
    <message>
        <location filename="geocsv_controller.py" line="282"/>
        <source>Couldn&apos;t automatically restore csv layer &quot;{}&quot;</source>
        <translation>Couldn&apos;t automatically restore csv layer &quot;{}&quot;</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="280"/>
        <source>GeoCSV Layer reconnected</source>
        <translation>GeoCSV layer reconnected</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="280"/>
        <source>Layer &quot;{}&quot; is successfully reconnected</source>
        <translation>Layer &quot;{}&quot; is successfully reconnected.</translation>
    </message>
</context>
<context>
    <name>VectorLayerController</name>
    <message>
        <location filename="geocsv_controller.py" line="294"/>
        <source>CSV File updated</source>
        <translation>CSV file updated</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="294"/>
        <source>Changes to layer &quot;{}&quot; successfully stored in csv file.</source>
        <translation>Changes to layer &quot;{}&quot; successfully stored in csv file.</translation>
    </message>
</context>
<context>
    <name>VectorLayerSaveConflictController</name>
    <message>
        <location filename="geocsv_controller.py" line="370"/>
        <source>Invalid path</source>
        <translation>Invalid path</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="370"/>
        <source>An error occured while trying to save file on new location. Please try again.</source>
        <translation>An error occured while trying to save file to new location. Please try again.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="319"/>
        <source>CSVT file could not be updated</source>
        <translation>CSVT file could not be updated</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="319"/>
        <source>An error occured while trying to update the CSVT file according to the new attribute types. Please update the csvt file manually.</source>
        <translation>An error occured while trying to update the CSVT file according to the new attribute types. Please update the csvt file manually.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="314"/>
        <source>Error while updating attributes happend</source>
        <translation>Error while updating attributes happend</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="314"/>
        <source>An error occured while trying to update the attributes list. Nothing has been stored on disk.</source>
        <translation>An error occured while trying to update the attributes list. Nothing has been stored on disk.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="324"/>
        <source>Geometry index violation</source>
        <translation>Geometry index violation</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="324"/>
        <source>You tried to delete an attribute which is providing geometry information. The change will not be saved to disk.</source>
        <translation>You deleted an attribute that was providing geometry information. This can lead to unpredictable problems. Please exit edit mode without saving your changes.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="368"/>
        <source>CSV File updated</source>
        <translation>CSV file updated</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="368"/>
        <source>Changes to layer &quot;{}&quot; successfully stored in csv file.</source>
        <translation>Changes to layer &quot;{}&quot; successfully stored in csv file.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="361"/>
        <source>Save File</source>
        <translation>Save File</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="361"/>
        <source>Files (*.csv *.tsv *.*)</source>
        <translation>Files (*.csv *.tsv *.*)</translation>
    </message>
</context>
<context>
    <name>geocsvDialogNew</name>
    <message>
        <location filename="geocsv_dialog_new.ui" line="14"/>
        <source>GeoCSV Editor</source>
        <translation>Add new GeoCSV layer</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_conflict.ui" line="29"/>
        <source>Retry</source>
        <translation>Retry</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="74"/>
        <source>Cancel</source>
        <translation>Cancel</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_conflict.ui" line="43"/>
        <source>Save As</source>
        <translation>Save As...</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_conflict.ui" line="59"/>
        <source>Error: The csv file could not be saved. File Lock?</source>
        <translation>The action can&apos;t be completed because the file couldn&apos;t be found or is currently in use by another program.</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="29"/>
        <source>Dateiname</source>
        <translation>Filename</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="52"/>
        <source>Durchsuchen...</source>
        <translation>Browse...</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="84"/>
        <source>Ok</source>
        <translation>OK</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="128"/>
        <source>WKT Field</source>
        <translation>WKT Field</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="142"/>
        <source>Point (Easting/Northing)</source>
        <translation>Easting/Northing (Point)</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="161"/>
        <source>WKT</source>
        <translation>WKT (Point, LineString, Polygon)</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="206"/>
        <source>Easting Field</source>
        <translation>Easting Field</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="219"/>
        <source>Northing Field</source>
        <translation>Northing Field</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="248"/>
        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://giswiki.hsr.ch/Editable_GeoCSV#Help&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2b84d3;&quot;&gt;Help&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="276"/>
        <source>Convert</source>
        <translation>Convert</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="283"/>
        <source>Character set:</source>
        <translation>Character set:</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="299"/>
        <source>The csv File is not in UTF-8. Please convert it first
 (CAUTION: the original file will be overwritten):</source>
        <translation>The csv file is not UTF-8 encoded. Please convert it first. (CAUTION: The original file will be overwritten. A backup is being created.)</translation>
    </message>
</context>
</TS>
